packages/@react-aria/table/docs/TableAnatomy.svg

```diff
@@ -0,0 +1,91 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="842" height="342" style="max-height: 342px; background: var(--anatomy-gray-100); padding: 32px; margin-bottom: 16px; width: calc(100% - 64px); border-radius: 4px;" viewBox="0 0 842 342">
```

Comment:
Right now the table anatomy diagram doesn't show the white background for the table body since it covers the lines for the dividers. This would be fixable by moving the `rect`s that make up the divider lines below `rectangle-33` but the svg transformer seems to modify the DOM order of the svg elements automatically. Digging into why right now (seems to be the parcel svg react transformer?), but IMO shouldn't block the PR can be handled in followup